### PR TITLE
Produce TinyV2 mappings for 1.14.4

### PR DESCRIPTION
**Pull request by natanfudge**
This is the same build.gradle file as the default branch.

### CODE REVIEW

**Comment by liach** [comment]
I suggest using the build.gradle at fb663d6f1dab98ccada33cb2f93dd53fc7d1dd93

**Comment by natanfudge** [comment]
oh right modmuss changed his setup

**Comment by modmuss50** [comment]
Yeah, it doesnt make too much diffrence here but keeping them the same might be nice.

Lets make sure its all working well in 1.15 before merging this as well. I want to merge this as it requires changing the order on the maven manually.

File: build.gradle [hunk 1]
```diff
@@ -445,57 +457,56 @@ clean.doFirst {
...
-			"enigma",
-			file("mappings/").getAbsolutePath(),
-			"right"
+				"tiny",
+				composeInput.getAbsolutePath(),
+				"enigma",
+				file("mappings_official/").getAbsolutePath(),
+				"enigma",
+				file("mappings/").getAbsolutePath(),
+				"right"
```
Comment (Sollace) [inline build.gradle:504]: There's a lot of formatting changes in here. IMO I'd try to keep it the same as it was for a cleaner PR.

Comment (natanfudge) [reply build.gradle:504]: it's fineee

**Comment by modmuss50** [comment]
Before merging this I do want to check that is isnt changing the auto generated names? As that may have been an issue on the main branch

**Comment by natanfudge** [comment]
I'm going to add a flag to stitch to be able to toggle that.

**Comment by modmuss50** [comment]
Sounds good, sadly if it was a bug we need to keep the bug for the older versions.

**Comment by natanfudge** [comment]
Well, you _can_ decide that it's desired behavior that it will only insert _unmapped_ names.

**Comment by modmuss50** [comment]
No, the dev shoudlnt have this option imo as source code for x version of yarn should work in someone else's dev env with the same version. For already released versions it needs to be kept the same. I dont see an issue in changing it from the current snapshot onwards though.

**Comment by natanfudge** [comment]
No, not the dev, **you**.

**Comment by liach** [comment]
you need to update stitch in another place below as well.

File: build.gradle (hunk 1 above)
Comment (modmuss50) [reply build.gradle:504]: Slighly annoying, but yeah dont worry.

**Review by modmuss50 — APPROVED** [review]
Im happy with this, v2 seems to be working well in 1.15 so this should hopefully be all thats needed for 1.14

**Review by Player3324 — APPROVED** [review]

**Pull request merged**